docs/GettingStarted.md

```diff
@@ -53,15 +56,15 @@ SnappyData makes the following contributions to deliver a unified and optimized
 * **Integrating an operational in-memory data store with Spark’s computational model**: A number of extensions are introduced to fuse our runtime with that of Spark. Spark executors run in the same process space as our store’s execution threads, sharing the same pool of memory. When Spark executes tasks in a partitioned manner, it is designed to keep all the available CPU cores busy. <br/> This design is extended by allowing low latency and fine- grained operations to interleave and get higher priority, without involving the scheduler. Furthermore, to support high concurrency, the runtime is extended with a “Job Server” that decouples applications from data servers, operating much in the same way as a traditional database, whereby the state is shared across many clients and applications. <br/>
 
 * **Unified API for OLAP, OLTP, and Streaming**: Spark builds on a common set of abstractions to provide a rich API for a diverse range of applications, such as MapReduce, Machine learning, stream processing, and SQL.
-While Spark deserves much of the credit for being the first of its kind to offer a unified API, SnappyData further extends its API to: 
+SnappyData extends Spark’s unified API: 
 	
 	* Allow for OLTP operations, for example, transactions and mutations (inserts/updates/deletions) on tables 
```

Comment:
add: A unique addition of SnappyData is ability to mutate (all of inserts/updates/deletes) even column format tables efficiently without much change in query performance profile.